cli/src/main/java/de/jplag/cli/JPlagMode.java

```diff
@@ -0,0 +1,19 @@
+package de.jplag.cli;
+
+/**
+ * The mode JPlag runs in. This influences which steps JPlag will execute
+ */
+public enum JPlagMode {
+    /**
+     * Only run JPlag and create a results.zip
+     */
+    RUN,
+    /**
+     * Only start the report viewer
+     */
+    VIEWER,
```

Comment:
Minor inconsistency: Should we call this `VIEW` to be consistent with `RUN` and `RUN_AND_VIEW`?